tutorials/animation/cutout_animation.rst

```diff
@@ -6,47 +6,46 @@ Cutout animation
 What is it?
 ~~~~~~~~~~~
 
-Cut-out is a technique of animating in 2D where pieces of paper (or
-similar material) are cut in special shapes and laid one over the other.
-The papers are animated and photographed, frame by frame using a stop
-motion technique (more info
-`here <https://en.wikipedia.org/wiki/Cutout_animation>`__).
-
-With the advent of the digital age, this technique became possible using
-computers, which resulted in an increased amount of animation TV shows
-using digital Cut-out. Notable examples are `South
-Park <https://en.wikipedia.org/wiki/South_Park>`__ or `Jake and the Never
-Land
-Pirates <https://en.wikipedia.org/wiki/Jake_and_the_Never_Land_Pirates>`__
-.
+Traditionally, `cutout animation <https://en.wikipedia.org/wiki/Cutout_animation>`__
```

Comment:
This was an edit "for clarity" in d12a5c81, so I assume it does not introduce 3.2-specific concepts.